server/app/assets/javascripts/enumerator.ts

```diff
@@ -174,14 +187,18 @@ function addEnumeratorListeners() {
   const mutationObserver = new MutationObserver((records: MutationRecord[]) => {
     for (const record of records) {
       for (const newNode of Array.from(record.addedNodes)) {
-        const newInputs = Array.from(
-          (<Element>newNode).querySelectorAll('input'),
-        )
-        newInputs.forEach((newInput) => {
-          newInput.addEventListener('input', () => {
-            maybeToggleEnumeratorAddButton(enumeratorQuestion)
+        // changes to the label and button texts trigger the mutationObserver which results in an error
+        // this if statement protects against that case
+        if (typeof (<Element>newNode).querySelectorAll !== 'undefined') {
```

Comment:
wondering if this explicit call to not equal is needed. I have seen if(element) --> automatically checks if the element is not null. So something like if(<Element>newNode).querySelectorAll)) might work here

Reply:
good callout! i thought i had tried that and it didn't work, but i just tried again and it works fine so i updated it.